src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
I think this is local files and can be added in `.gitignore`.

Reply:
You'll need this file ... so if we are adding it to `.gitignore` then we have to give clear Instruction for adding this file, in our README.